internal/clientio/resp.go

```diff
@@ -152,17 +152,42 @@ func encodeString(v string) []byte {
 	return []byte(fmt.Sprintf("$%d\r\n%s\r\n", len(v), v))
 }
 
+// encodeBool encodes bool as simple strings
+func encodeBool(v bool) []byte {
+	var val string
+	switch v {
+	case true:
+		val = "true"
+	case false:
+		val = "false"
+	}
+	return []byte(fmt.Sprintf("+%v\r\n", val))
+}
```

Comment:
Maybe this can be replaced by the following snippet. And in that case it doesn't need to be a separate function, it can be written directly inside the switch case of the overall `Encode` function.
```suggestion
func encodeBool(v bool) []byte {
	return []byte(fmt.Sprintf("+%t\r\n", v))
}
```

Reply:
This works. Thanks for suggestion. Updated.